chore(docs): improve description of access rights for integration projects (#184)
camunda/camunda-docs-cawemo: content/user-guide/projects.md

- **Commenter:** the user cannot edit folders or diagrams nor invite users, but can leave [comments](../diagrams#comments) on each diagram.
- **Viewer:** the user cannot edit folders or diagrams nor leave comments, but only view diagrams.

The **Project Admin** is the only user entitled to invite and manage user roles.

On the Home Page you are able to easily identify who has access to the project:

{{<img src="../default-project.png" title="Project created from Cawemo - collaborators visibility" width="150%" height="150%">}}

{{< note class="info" >}}
When there are more than a handful of users, you can hover over the circle with a number on it to view the names of all users that have access.
{{< /note >}}

### Projects used by integrations

Projects used by integrations do not allow you to manage access rights.
All [members of your organization](../settings#managing-members-admin-only) automatically gain _read-only_ access to your catalog, engine and modeler projects, meaning that they can see your files, but not edit them.

> On the Home Page you are able to easily identify who has access to the project. In this case: **Everyone** in the organization has access:

Reviewer: ```suggestion
On the Home Page you are able to easily identify who has access to the project. 
In this case **everyone** in the organization has access:
```

Author: Changes done in https://github.com/camunda/camunda-docs-cawemo/pull/184/commits/5eb039b821aa7fa664be1bdb5c8e1f8d88433c2b

- **Commenter:** the user cannot edit folders or diagrams nor invite users, but can leave [comments](../diagrams#comments) on each diagram.
- **Viewer:** the user cannot edit folders or diagrams nor leave comments, but only view diagrams.

The **Project Admin** is the only user entitled to invite and manage user roles.

> In your Home Page you are able to easily identify all people that have access to a particular project.

Reviewer: ```suggestion
On the home page you are able to easily identify all people that have access to a particular project.
```

{{<img src="../default-project.png" title="Project created from Cawemo - collaborators visibility" width="150%" height="150%">}}

{{< note class="info" >}}
When there are more than a handful of users, you can hover over the circle with a number on it to view the names of all users that have access.
{{< /note >}}

### Projects created via integrations

Projects created via our integrations do not allow you to manage access rights.
All [members of your organization](../settings#managing-members-admin-only) automatically gain _read-only_ access to your catalog, engine and modeler projects, meaning that they can see your files, but not edit them.

> In your Home Page you are able to easily identify all people that have access to a particular project.

Reviewer: I'd completely remove this sentence as it is just a copy from above and might be confusing as everyone has access to these kind of projects.

Author: I see multiple sentences highlighted, could you please clarify to exactly which one you are referring to?

Author: Maybe:

```suggestion
### Projects used by integrations

Projects used by integrations do not allow you to manage access rights.
All [members of your organization](../settings#managing-members-admin-only) automatically gain _read-only_ access to your catalog, engine and modeler projects, meaning that they can see your files, but not edit them.

In your Home Page you are able to easily identify who has access to a project:
{{<img src="../integration-projects.png" title="Projects created via integrations - collaborators visibility" width="150%" height="150%">}}

{{< note class="info" >}}
In this case: **Everyone** has access to these projects.
{{< /note >}}
```

Is this clearer?

Reviewer: It's clearer now from my perspective but I'm still not a fan of putting the sentence `In this case: **Everyone** has access to these projects.` into a note but also not an expert on this topic, a technical writer should probably have another look.

Author: I can just get it out of the note and put it in the paragraph before the image.

{{<img src="../integration-projects.png" title="Projects created via integrations - collaborators visibility" width="150%" height="150%">}}

{{< note class="info" >}}
In this case: **Everyone** has access to these projects.
{{< /note >}}

> 

Reviewer: Let's only insert one blank line in between the sections
